src/static/less/pages/homepage.less

```diff
@@ -658,6 +667,14 @@ section.video {
     }
   }
 
+  #about-the-rule {
+    h2 {
+      background-image: url( '../img/book-isocon.png');
```

Comment:
Really odd!?!

I see the correct icon, the book, in both Chrome and Firefox. But, made the change and appears all is well. Had to update the icon itself too.